diffkemp/simpll/simpll_build.py

```diff
@@ -37,15 +50,16 @@ def get_c_declarations(header_filename):
 llvm_libs = list(filter(lambda x: x != "",
                         llvm_libs.decode("ascii").strip().split()))
 
-simpll_link_arg = "-L{}/diffkemp/simpll".format(get_simpll_build_dir())
+simpll_link_arg = f"-L{utils.get_simpll_build_dir()}/diffkemp/simpll"
+
+include_path = f'{root_dir}/diffkemp/simpll'
 
 ffibuilder.set_source(
-    "diffkemp.simpll._simpll", '#include <library/FFI.h>',
+    "_simpll", '#include <library/FFI.h>',
```

Comment:
Maybe a note in a commit message would be useful that previously when running `pip install -e .` the `CFFI _simpll...so` file was saved in `./diffkemp/simpll` (respectively in `site-packages/diffkemp/simpll` when running `pip install .`) subdirectory but now it is saved in DiffKemp root directory (respectively directly in `site-packages` directory).

Reply:
the `.so` file is not saved in the root directory, it is saved in `site-packages` (somewhere inside, this detail is now not important) or it is going to be saved in `BUILDDIR/diffkemp/simpll`, this is the reason why we need the code snippet to which you have commented below.

But making the commit messages better is a good point, I will look into it.

Reply:
> this is the reason why we need the code snippet to which you have commented below.

I am not sure about this, because in the `build/bin/diffkemp` (respectively in its template) you are adding `${CMAKE_BINARY_DIR}/diffkemp/simpll` directory to system path, so I think that even if the `.so` file is saved in `BUILDDIR/diffkemp/simpll` the `_simpll = __import__('_simpll')` import is used and not the `_simpll = __import__('diffkemp.simpll._simpll')`. I think that the second import would be used if `${CMAKE_BINARY_DIR}` was added to the system path without the `diffkemp/simpll` subdirectory, which was done previously.